dts/arm/st/f1/stm32f100Xe.dtsi

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+ *
+ * SoC device tree include for STM32F100xB SoCs
```

Comment:
STM32F100xE

Reply:
Ouch, embarrassing error. Fixed.

Man, you are patient :-) Much respect.